### PR TITLE
Add options to ease checks

### PR DESCRIPTION
**Pull request by take-cheeze**

### CODE REVIEW

**Comment by take-cheeze** [comment]
ChainerCI, test this please!

File: tools/run_onnx.cc [hunk 1]
```diff
@@ -414,7 +414,7 @@ class ModelRunner {
     std::vector<std::string> backprop_ins_;
 };
 
-void VerifyOutputs(const InOuts& outputs, const TestCase& test_case, const cmdline::parser& args, bool strict_check) {
+void VerifyOutputs(const InOuts& outputs, const TestCase& test_case, const cmdline::parser& args, bool strict_check, bool show_diff) {
```
Comment (shinh) [inline tools/run_onnx.cc]: My bad, I think `strict_check` wasn't a good name. How about renaming this to `check_values`?

File: tools/run_onnx.cc [hunk 2]
```diff
@@ -534,6 +537,8 @@ void RunMain(const std::vector<std::string>& argv) {
     args.add<double>("atol", '\0', "atol of AllClose", false, 1e-6);
     args.add("equal_nan", '\0', "Treats NaN equal");
     args.add("no_catch", '\0', "Do not catch the exception in ChxVM for better GDB experience");
+    args.add("no_strict_check", '\0', "Disable strict check of node output");
```
Comment (shinh) [inline tools/run_onnx.cc]: `no_check_values`?

File: tools/run_onnx.cc [hunk 3]
```diff
@@ -534,6 +537,8 @@ void RunMain(const std::vector<std::string>& argv) {
...
+    args.add("show_diff", '\0', "Show diff of strict check without stopping");
```
Comment (shinh) [inline tools/run_onnx.cc]: How about `always_show_diff`? Also, how about "Show diff even when value check is skipped" for the help message?

**Pull request merged**